### ISSUE TITLE
ICD telemetry subscription fields missing from Metadata

### ISSUE THREAD

**Issue by zonca** [oirlab/iris_icd_tool#1]
I am comparing `subscribe-model.conf` https://github.com/tmt-icd/IRIS-Model-Files/blob/master/drs/drs-assembly/subscribe-model.conf
and "Metadata_IRIS_new" https://docs.google.com/spreadsheets/d/1BZ7tCI6w8wkjrauZqGwZbrWzp1S_D6Nfi8AR2NSwiM0/edit?pli=1#gid=2146550726

I was expecting to find all the fields of `subscribe-model.conf` into "Metadata_IRIS_new", but there are several missing:

```
Missing Pandas(Index=2, subsystem='TCS', component='cmNFIRAOS', name='ngsTargetInfo', requiredRate=4.0, usage='Proposed item in which TCS publishes ICRS RA, Dec and proper motions, names, and colors (what system?) for the NGS WFS target. FITS=???')
Missing Pandas(Index=5, subsystem='TCS', component='cmNFIRAOS', name='atmDispersion', requiredRate=nan, usage=nan)
Missing Pandas(Index=6, subsystem='TCS', component='cmNFIRAOS', name='ngsPos', requiredRate=nan, usage=nan)
Missing Pandas(Index=10, subsystem='TCS', component='cmIRIS', name='oiwfs1AtmDispersion', requiredRate=nan, usage=nan)
Missing Pandas(Index=11, subsystem='TCS', component='cmIRIS', name='oiwfs2AtmDispersion', requiredRate=nan, usage=nan)
Missing Pandas(Index=12, subsystem='TCS', component='cmIRIS', name='oiwfs3AtmDispersion', requiredRate=nan, usage=nan)
Missing Pandas(Index=18, subsystem='TCS', component='TCS MCS Assembly', name='TCS Telescope Position', requiredRate=nan, usage=nan)
Missing Pandas(Index=20, subsystem='TCS', component='PointingKernelAssembly', name='CurrentPosition', requiredRate=nan, usage=nan)
Missing Pandas(Index=26, subsystem='NFIRAOS', component='rtc', name='lgsState', requiredRate=nan, usage=nan)
Missing Pandas(Index=28, subsystem='NFIRAOS', component='rtc', name='oiwfsImage', requiredRate=nan, usage=nan)
Missing Pandas(Index=29, subsystem='NFIRAOS', component='rtc', name='pwfsState', requiredRate=nan, usage=nan)
Missing Pandas(Index=31, subsystem='NFIRAOS', component='ssNgs', name='current', requiredRate=nan, usage=nan)
Missing Pandas(Index=35, subsystem='NFIRAOS', component='lgsTrombone', name='current', requiredRate=nan, usage=nan)
Missing Pandas(Index=38, subsystem='IRIS', component='imager.odgw', name='current', requiredRate=nan, usage=nan)
Missing Pandas(Index=41, subsystem='IRIS', component='imager.filter', name='dark', requiredRate=nan, usage=nan)
Missing Pandas(Index=42, subsystem='IRIS', component='imager.filter', name='current', requiredRate=nan, usage=nan)
Missing Pandas(Index=49, subsystem='IRIS', component='imager.adc', name='odgwShift', requiredRate=nan, usage=nan)
Missing Pandas(Index=50, subsystem='IRIS', component='imager.adc', name='PRISM_state', requiredRate=nan, usage=nan)
Missing Pandas(Index=53, subsystem='IRIS', component='imager.detector', name='exposureState', requiredRate=nan, usage=nan)
Missing Pandas(Index=54, subsystem='IRIS', component='imager.detector', name='startExposure', requiredRate=nan, usage=nan)
Missing Pandas(Index=55, subsystem='IRIS', component='imager.detector', name='readoutCompleted', requiredRate=nan, usage=nan)
Missing Pandas(Index=56, subsystem='IRIS', component='imager.detector', name='endExposure', requiredRate=nan, usage=nan)
Missing Pandas(Index=57, subsystem='IRIS', component='imager.detector', name='aborting', requiredRate=nan, usage=nan)
Missing Pandas(Index=58, subsystem='IRIS', component='imager.detector', name='aborted', requiredRate=nan, usage=nan)
Missing Pandas(Index=59, subsystem='IRIS', component='imager.detector', name='readoutFailed', requiredRate=nan, usage=nan)
Missing Pandas(Index=64, subsystem='IRIS', component='is', name='aborting', requiredRate=nan, usage=nan)
Missing Pandas(Index=69, subsystem='IRIS', component='is', name='startPrepare', requiredRate=nan, usage=nan)
Missing Pandas(Index=70, subsystem='IRIS', component='is', name='startDataAcquisition', requiredRate=nan, usage=nan)
Missing Pandas(Index=71, subsystem='IRIS', component='is', name='endDataAcquisition', requiredRate=nan, usage=nan)
Missing Pandas(Index=72, subsystem='IRIS', component='is', name='startReadout', requiredRate=nan, usage=nan)
Missing Pandas(Index=73, subsystem='IRIS', component='is', name='endReadout', requiredRate=nan, usage=nan)
Missing Pandas(Index=74, subsystem='IRIS', component='is', name='startDatasetWrite', requiredRate=nan, usage=nan)
Missing Pandas(Index=75, subsystem='IRIS', component='is', name='endDatasetWrite', requiredRate=nan, usage=nan)
Missing Pandas(Index=76, subsystem='IRIS', component='is', name='observeComplete', requiredRate=nan, usage=nan)
Missing Pandas(Index=77, subsystem='IRIS', component='is', name='observationId', requiredRate=nan, usage=nan)
Missing Pandas(Index=78, subsystem='IRIS', component='is', name='observerKeywords', requiredRate=nan, usage=nan)
Missing Pandas(Index=79, subsystem='IRIS', component='is', name='observationBlockKeywords', requiredRate=nan, usage=nan)
Missing Pandas(Index=80, subsystem='IRIS', component='is', name='scriptName', requiredRate=nan, usage=nan)
Missing Pandas(Index=96, subsystem='IRIS', component='ifs.detector', name='exposureState', requiredRate=nan, usage=nan)
Missing Pandas(Index=97, subsystem='IRIS', component='ifs.detector', name='startExposure', requiredRate=nan, usage=nan)
Missing Pandas(Index=98, subsystem='IRIS', component='ifs.detector', name='readoutCompleted', requiredRate=nan, usage=nan)
Missing Pandas(Index=99, subsystem='IRIS', component='ifs.detector', name='endExposure', requiredRate=nan, usage=nan)
Missing Pandas(Index=100, subsystem='IRIS', component='ifs.detector', name='aborting', requiredRate=nan, usage=nan)
Missing Pandas(Index=101, subsystem='IRIS', component='ifs.detector', name='aborted', requiredRate=nan, usage=nan)
Missing Pandas(Index=102, subsystem='IRIS', component='ifs.detector', name='readoutFailed', requiredRate=nan, usage=nan)
```

Just as an example the first field `ngsTargetInfo`, in Metadata_IRIS_new is in `cmIRIS` while in the ICD is in `cmNFIRAOS`.

@arunsurya77 can you fix them or is there a workaround I can use?

**Comment by zonca** [comment]
see notebook I used: https://gist.github.com/e0361f4d0b1954dd6c2ac47f9820a35e

**Comment by arunsurya77** [comment]
Hi Andrea ,
I had to remove some of the entries from metadata table that are events we
subscribe to but are not required as metadata. These are events regarding
the status of readout ,exposure etc which we will need to know as DRS and
is published as events.
But they don't go into metadata.

The cmIRIS is a mistake for that entry. I have corrected this in the table.
Also, have to keep in mind that not all subscriptions of subscribe.conf
have an event. Some are place holders that we require and are currently
negotiating ex with TCS.

Cheers
Arun
